menu_button.h

```diff
@@ -3,37 +3,40 @@
 
 #include <SFML/Graphics.hpp>
 #include "color.h"
+#include "coordinate.h"
 
 #include <cassert>
 #include <string>
 
 class menu_button
 {
 public:
-  menu_button(std::string name,
+  menu_button(std::string label,

```

Comment:
Why not `const std::string& label`? You may even be correct (but I keep the why a secret for now :-) ), I mostly check up on the reasoning here :-)

Reply:
It's just a variable used to initialise the value of the corresponding member function. I guess if we made this `const` the member variable would inherit from this and we wouldn't be able to change it anymore? Which is not desirable